eng/pipelines/mirror-repos.yml

```diff
@@ -19,9 +19,16 @@ jobs:
 
           Azure/azure-rest-api-specs:
```

Comment:
Do you want to explicitly specify main here as the SourceBranch? If you don't it will just do the default branch in the repo which currently is master but when we move we will later be main.

Reply:
No I current don't want to explicitly specify main and leave it as default branch is fine.